internal/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bazel/typescript",
+  "description": "Custom TypeScript compiler for hosting on Bazel",
+  "homepage": "https://github.com/bazelbuild/rules_typescript",
+  "license": "Apache-2.0",
+  "version": "0.0.0-PLACEHOLDER",
+  "keywords": [
```

Comment:
needs a "typings" field so my downstream import of `@bazel/typescript` can work (no index.d.ts in the package)

Reply:
there is a .d.ts file for each compiled .ts file in the npm_package. do you mean to have a central index.d.ts then that would include all of them?

Reply:
Added "typings": "./tsc_wrapped/index.d.ts" to @bazel/typescript package.json